conftest.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
Can we move this to a sub-directory? move it to "hub" at least.

Reply:
no i tried and it doesn't work. conftest needs to be at the same level as setup.py

Reply:
okay